### PR TITLE
Delet button

### PR DESCRIPTION
**Pull request by chris-kuehn**

### CODE REVIEW

File: components/Navigation.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ const StyledNavigationLink = styled(Link)`
   letter-spacing: 0.75px;
   &:hover {
     color: var(--primary-color);
+  }
+
+  &:active {
+    /* color: var(--off-white); */
```
Comment (mbosselmann) [inline components/Navigation.js:71]: More outcommented code that can be removed. :)

File: pages/gallery.js [hunk 2]
```diff
@@ -28,8 +41,24 @@ export default function HomePage({ listOfArtworks }) {
 const StyledListItem = styled.li`
   list-style: none;
   padding: 0;
+  margin-bottom: 3rem;
 `;
 
 const StyledUl = styled.ul`
```
Comment (mbosselmann) [inline pages/gallery.js:47]: A better name would be `StyledList`. :)

File: pages/gallery.js [hunk 3]
```diff
@@ -28,8 +41,24 @@ export default function HomePage({ listOfArtworks }) {
...
   padding: 0;
 `;
+
+const StyledDeletButton = styled.button`
```
Comment (mbosselmann) [inline pages/gallery.js:51]: A typo: `StyledDeleteButton` :)

File: styles.js [hunk 4]
```diff
@@ -54,4 +64,10 @@ button {
   font-family: ${robotoCondensed.style.fontFamily};
   
 }
+button {
+transition: background-color 250ms ease-in-out;}
+a {
+  transition-property: color, text-decoration;
+transition-duration: 500ms;
+transition-timing-function: ease-in-out;}
 `;
```
Comment (mbosselmann) [inline styles.js:73]: I think the formatting of your code went wrong or? 🤔

**Review by mbosselmann — COMMENTED** [review]
Hi Christian,
well done! 👍 I left some comments for you, but nothing that hinders me to approve your PR. :)

File: components/Inputs.js [hunk 5]
```diff
@@ -8,7 +8,22 @@ export default function InputsOne({
   handleOutlineChange,
   handleVariation,
   handleListOfArtworks,
+  artworkId,
+  setArtworkId,
+  setCanvasParameter,
 }) {
+  // function handleAddArtwork() {
+  //   setArtworkId(artworkId + 1);
+  // }
```
[the remark this answers is not in the document]
Comment (chris-kuehn) [reply components/Inputs.js:17]: upps I did it again .... ;)

Deleted!

**Pull request merged**